crates/core_simd/src/vector.rs

```diff
@@ -863,7 +864,7 @@ where
 {
     type Error = core::array::TryFromSliceError;
 
-    fn try_from(slice: &mut [T]) -> Result<Self, Self::Error> {
+    fn try_from(slice: &mut [T]) -> Result<Self, core::array::TryFromSliceError> {
```

Comment:
why not just leave it as `Self::Error` -- it's shorter and imho better (sorry icr the word I wanted insead of better -- basically imho more aligned with norms of rust code)

Reply:
idiomatic? :)

Reply:
not what I was thinking of, but good enough!

Reply:
@programmerjake because `x check` screams at me when I do.

Reply:
oh, ok....